Libraries/Animated/Animated.js

```diff
@@ -9,38 +9,26 @@
  */
 
 import Platform from '../Utilities/Platform';
-import typeof AnimatedFlatList from './components/AnimatedFlatList';
-import typeof AnimatedImage from './components/AnimatedImage';
-import typeof AnimatedScrollView from './components/AnimatedScrollView';
-import typeof AnimatedSectionList from './components/AnimatedSectionList';
-import typeof AnimatedText from './components/AnimatedText';
-import typeof AnimatedView from './components/AnimatedView';
+import AnimatedFlatList from './components/AnimatedFlatList';
+import AnimatedImage from './components/AnimatedImage';
+import AnimatedScrollView from './components/AnimatedScrollView';
+import AnimatedSectionList from './components/AnimatedSectionList';
+import AnimatedText from './components/AnimatedText';
+import AnimatedView from './components/AnimatedView';
 
-import * as AnimatedMock from './AnimatedMock';
-import * as AnimatedImplementation from './AnimatedImplementation';
+import AnimatedMock from './AnimatedMock';
+import AnimatedImplementation from './AnimatedImplementation';
 
 const Animated = ((Platform.isTesting
   ? AnimatedMock
   : AnimatedImplementation): typeof AnimatedMock);
 
 module.exports = {
-  get FlatList(): AnimatedFlatList {
-    return require('./components/AnimatedFlatList');
-  },
-  get Image(): AnimatedImage {
-    return require('./components/AnimatedImage');
-  },
-  get ScrollView(): AnimatedScrollView {
-    return require('./components/AnimatedScrollView');
-  },
-  get SectionList(): AnimatedSectionList {
-    return require('./components/AnimatedSectionList');
-  },
-  get Text(): AnimatedText {
-    return require('./components/AnimatedText');
-  },
-  get View(): AnimatedView {
-    return require('./components/AnimatedView');
-  },
+  FlatList: AnimatedFlatList,
+  Image: AnimatedImage,
+  ScrollView: AnimatedScrollView,
+  SectionList: AnimatedSectionList,
+  Text: AnimatedText,
+  View: AnimatedView,
   ...Animated,
 };
```

Comment:
I think we'll want to keep this, as it enables lazy initialization of these modules (if you don't have the inline-requires transform enabled)

Reply:
Got it, I'll revert this

Reply:
@javache What do you mean by this `if you don't have the inline-requires transform enabled` ? Could you explain it a little bit more ?

Reply:
Reverted

Reply:
@TMaszko see https://reactnative.dev/docs/ram-bundles-inline-requires for details on inline-requires. It's an important transform we apply to improve bundle init time.

Reply:
I know that optimization technique but my question is why getters do enable lazy initialization when inline-requires transform is disabled? I thought it will work **only** when this transform is enabled ;)

Reply:
Ah, sorry for misunderstanding. In this case the `require` is manually inlined, and we would only invoke the module factory for that module once the getter is invoked. When using `import` syntax, that manual inlining is not possible.

 When the inline-requires transform is enabled, you get this behaviour for free (as long as you're using a getter in this case)